docker/setup-compose.yml

```diff
@@ -6,14 +6,14 @@
 #
 # then run:
 # docker-compose -f unit-tests-compose.yml up
-# 
+#
 # To clean up:
 # docker-compose -f unit-tests-compose.yml down
 
-version: '3'
+version: "3"
 services:
   site:
-    image: wintercdo/code-dot-org:0.7
+    image: codedotorg/code-dot-org
```

Comment:
We talked offline about whether this should be versioned. Have you had more thoughts about this?

Reply:
Right! just pushed a change doing that